ref: Improve JS SDK Component Tracking Documentation (#5619)
getsentry/sentry-docs: src/platforms/react-native/performance/instrumentation/automatic-instrumentation.mdx

### React Profiler

> We export the React Profiler from our React Native SDK as well, you can read more at [React Component Tracking](/platforms/javascript/guides/react/components/componenttracking/).

Reviewer: ```suggestion
We export the React Profiler from our React Native SDK as well. Learn more in [React Component Tracking](/platforms/javascript/guides/react/components/componenttracking/).
```
